osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/config/JasperWirelessAccess.java

```diff
@@ -8,44 +8,17 @@
  */
 package org.opensmartgridplatform.adapter.protocol.jasper.config;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
```

Comment:
AllArgsConstructor are not preferred, because change in order of attributes, changes constructor

Reply:
Replaced with coded all args constructor